frontend/pages/configurator.vue

```diff
@@ -638,12 +638,41 @@ const duplicateItem = (index) => {
   testItems.value.push(newItem); // Neues Item zu `testItems` hinzufügen
 };
 
-const deleteItem = (index) => {
-  const itemId = testItems.value[index].id;
+const deleteItem = async (index) => {
+  const item = testItems.value[index];
+  const itemId = item.id;
+
+  // Prüfen, ob die Item-Config eine persistente ID aus der Datenbank hat
+  if (itemId && !item.isUnsaved) {
+    try {
+      const token = localStorage.getItem("token");
+      await $fetch(`http://localhost:8000/api/item_configs/${itemId}`, {
+        method: "DELETE",
+        headers: {
+          Authorization: `Bearer ${token}`,
+        },
+      });
+
+      // Erfolgsmeldung anzeigen
+      triggerAlert("success", "Item erfolgreich gelöscht!");
```

Comment:
Small thing: it would be great if this alert also comes if you delete one that is not yet saved to the db and is only deleted in the frontend. Currently, you sometimes get the alert and sometimes not depending on whether it is a Vorlage or not. I think this would be better if it is consistent (hope this doesn't get annoying :) )